Library/Homebrew/github_packages.rb

```diff
@@ -419,7 +440,7 @@ def upload_bottle(user, token, skopeo, formula_full_name, bottle_hash, keep_old:
       rescue ErrorDuringExecution
         retry_count += 1
         odie "Cannot perform an upload to registry after retrying multiple times!" if retry_count >= 5
-        sleep 5*retry_count
+        sleep 5 ** retry_count
```

Comment:
```suggestion
        sleep 2 ** retry_count
```

maybe? can tweak post-merge if it seems excessive